src/main/resources/ome/services/startup.xml

```diff
@@ -22,6 +22,26 @@
   HOOKS : These beans are used to provide startup and shutdown logic.
   </description>
 
+  <bean id="version.package" class="ome.system.UpgradeCheck"
+        factory-method="getPackage"/>
+  <bean id="omero.version" factory-bean="version.package"
+        factory-method="getImplementationVersion"/>
+  <bean class="org.springframework.beans.factory.config.MethodInvokingFactoryBean">
+        <property name="targetObject">
+            <bean class="org.springframework.beans.factory.config.MethodInvokingFactoryBean">
+                <property name="targetClass" value="java.lang.System" />
+                <property name="targetMethod" value="getProperties" />
+            </bean>
+        </property>
+        <property name="targetMethod" value="setProperty" />
+        <property name="arguments">
+            <list>
+                <value>omero.verison</value>
```

Comment:
omero.version

Reply:
Hmmm.... thanks. That suggests though that another version is getting picked up somewhere despite my seeing:
```
$ bin/omero version
5.5.0-m3-11-26697c4c57-dirty-ice36-b1

$ bin/omero shell --login
...
In [1]: client.sf.getConfigService().getVersion()
Out[1]: '5.5.0'

In [2]: client.sf.getConfigService().getConfigValue("omero.version")
Out[2]: '5.5.0-m3-11-26697c4c57-dirty-ice36-b1'
```

Reply:
* I rebuilt and published locally omero-server
* Ran ``./build.py clean && ./build.py``
* error when starting openmicrospy:
```
	at org.springframework.beans.factory.access.SingletonBeanFactoryLocator.useBeanFactory(SingletonBeanFactoryLocator.java:404) ~[spring-beans.jar:4.3.22.RELEASE]
	at ome.system.OmeroContext.getInstance(OmeroContext.java:202) ~[omero-common.jar:5.5.0-m4]
	at ome.services.blitz.Entry.start(Entry.java:189) [omero-blitz.jar:5.5.0-SNAPSHOT]
	at ome.services.blitz.Entry.main(Entry.java:146) [omero-blitz.jar:5.5.0-SNAPSHOT]
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'OMERO.blitz' defined in URL [jar:file:/Users/jmarie/Documents/git-repo/openmicroscopy/dist/lib/server/omero-blitz.jar!/beanRefContext.xml]: Cannot resolve reference to bean 'ome.server' while setting constructor argument; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'ome.server' defined in URL [jar:file:/Users/jmarie/Documents/git-repo/openmicroscopy/dist/lib/server/omero-server.jar!/beanRefContext.xml]: Bean instantiation via constructor failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [ome.system.OmeroContext]: Constructor threw exception; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'version.package' defined in class path resource [ome/services/startup.xml]: No matching factory method found: factory method 'getPackage()'. Check that a method with the specified name exists and that it is static.	at org.springframework.beans.factory.access.SingletonBeanFactoryLocator.useBeanFactory(SingletonBeanFactoryLocator.java:404) ~[spring-beans.jar:4.3.22.RELEASE]
	at ome.system.OmeroContext.getInstance(OmeroContext.java:202) ~[omero-common.jar:5.5.0-m4]
	at ome.services.blitz.Entry.start(Entry.java:189) [omero-blitz.jar:5.5.0-SNAPSHOT]
	at ome.services.blitz.Entry.main(Entry.java:146) [omero-blitz.jar:5.5.0-SNAPSHOT]
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'OMERO.blitz' defined in URL [jar:file:/Users/jmarie/Documents/git-repo/openmicroscopy/dist/lib/server/omero-blitz.jar!/beanRefContext.xml]: Cannot resolve reference to bean 'ome.server' while setting constructor argument; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'ome.server' defined in URL [jar:file:/Users/jmarie/Documents/git-repo/openmicroscopy/dist/lib/server/omero-server.jar!/beanRefContext.xml]: Bean instantiation via constructor failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [ome.system.OmeroContext]: Constructor threw exception; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'version.package' defined in class path resource [ome/services/startup.xml]: No matching factory method found: factory method 'getPackage()'. Check that a method with the specified name exists and that it is static.
```

Reply:
Yeah, hold off J-m. There's already the one issue from Mark. My assumption is that I'm picking up the wrong jar.